commands/calls.js

```diff
@@ -0,0 +1,45 @@
+'use strict'
+
+const co = require('co')
+const cli = require('heroku-cli-util')
+const pg = require('heroku-pg')
+const util = require('../lib/util')
+
+function * run (context, heroku) {
+  let db = yield pg.fetcher(heroku).database(context.app, context.args.database)
+
+  yield util.ensurePGStatStatement(db)
+
+  let truncatedQueryString = context.flags.truncate
+    ? 'CASE WHEN length(query) < 40 THEN query ELSE substr(query, 0, 38) || \'..\' END'
```

Comment:
Also I suppose that should be `<= 40`--there's no sense to truncating to 38 characters and adding two dots when that makes it 40 characters again.